modules/initramfs.nix

```diff
@@ -53,6 +53,11 @@ in
           nod /dev/mtdblock3 0600 0 0 b 31 3
           nod /dev/mtdblock4 0600 0 0 b 31 4
           nod /dev/mtdblock5 0600 0 0 b 31 5
+          nod /dev/mtdblock6 0600 0 0 b 31 5
+          nod /dev/mtdblock7 0600 0 0 b 31 5
+          nod /dev/mtdblock8 0600 0 0 b 31 5
+          nod /dev/mtdblock9 0600 0 0 b 31 5
+          nod /dev/mtdblock10 0600 0 0 b 31 5
           dir /target 0755 0 0
```

Comment:
am now using devtmpfs in initramfs (because of /dev/vda which doesn't have a static major number), so this is not needed any more